test/unit/api/test_api.py

```diff
@@ -153,8 +140,12 @@ def test_buckets_with_encryption(self):
             mode=EncryptionMode.SSE_B2,
             algorithm=EncryptionAlgorithm.AES256,
         )
-        no_encryption = EncryptionSetting(mode=EncryptionMode.NONE,)
-        unknown_encryption = EncryptionSetting(mode=EncryptionMode.UNKNOWN,)
+        no_encryption = EncryptionSetting(
+            mode=EncryptionMode.NONE,
+        )
+        unknown_encryption = EncryptionSetting(
+            mode=EncryptionMode.UNKNOWN,
+        )
```

Comment:
This change was made only because of the trailing comma in `EncryptionSetting(mode=EncryptionMode.UNKNOWN,)` - black treats a trailing comma as an indicator that this is a list where each element should be in a separate line.